pages/osx/netstat.md

```diff
@@ -21,3 +21,7 @@
 - List information continuously:
 
 `netstat -c`
+
+- Print the route table:
```

Comment:
```suggestion
- Print the routing table:
```

Reply:
I've personally never heard it referred to as a 'routing table', but Google (and Wikipedia) suggest that that term is more common. Updated.